src/pages/mining.md

```diff
@@ -126,6 +126,108 @@ In case you are running into issues or would like to see verbose logging, you ca
 BLOCKSTACK_DEBUG=1 stacks-node krypton
 ```
 
+## Running a miner in Windows
+
```

Comment:
are there any requirements that we should point out?

Reply:
Updated the requirements.